lib/upgrade/upgrade_supported.go

```diff
@@ -110,6 +110,29 @@ func FetchLatestReleases(releasesURL, current string) []Release {
 	}
 	resp.Body.Close()
 
+	for i, rel := range rels {
+		if len(rel.RuntimeReqs.Requirements) > 0 {
+			continue // requirements already filled in
+		}
+
+		// see if a compat.json is available, if so fill out the
+		// requirements
+		for _, asset := range rel.Assets {
+			if asset.Name == "compat.json" {
+				resp, err := insecureGet(asset.BrowserURL, current)
+				if err != nil {
+					l.Infoln("Fetching compat.json:", err)
```

Comment:
Maybe worth knowing which release failed (same for the error below):

```suggestion
					l.Infof("Fetching compat.json for %v: %v", rel.Tag, err)
```